webapp/src/components/admin_console_settings/rtcd_service_url/index.tsx

```diff
@@ -0,0 +1,258 @@
+import {
+    IDOnPremTrialError,
+    IDOnPremTrialSuccess,
+    OnPremTrialError,
+    OnPremTrialSuccess,
+} from 'src/components/admin_console_settings/rtcd_service_url/modals';
+import {requestOnPremTrialLicense} from 'src/actions';
+
+const RTCDServiceUrl = (props: CustomComponentProps) => {
+    const dispatch = useDispatch();
+    const restricted = useSelector(isOnPremNotEnterprise);
+    const cloud = useSelector(isCloud);
+    const stats = useSelector(adminStats);
+
+    const [currentRestricted, setCurrentRestricted] = useState(restricted);
+
+    const leftCol = 'col-sm-4';
+    const rightCol = 'col-sm-8';
+
+    // Webapp doesn't pass the placeholder setting.
+    const placeholder = 'https://rtcd.example.com';
+
+    const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+        props.onChange(props.id, e.target.value);
+    };
+
+    const requestLicense = async () => {
+        let users = 0;
+        if (stats && (typeof stats.TOTAL_USERS === 'number')) {
+            users = stats.TOTAL_USERS;
+        }
+        const requestedUsers = Math.max(users, 30);
+
+        const {error} = await requestOnPremTrialLicense(requestedUsers, true, true);
+
+        if (error) {
+            dispatch(modals.openModal({
+                modalId: IDOnPremTrialError,
+                dialogType: OnPremTrialError,
+            }));
+        } else {
+            dispatch(modals.openModal({
+                modalId: IDOnPremTrialSuccess,
+                dialogType: OnPremTrialSuccess,
+            }));
+            setCurrentRestricted(false);
+        }
+    };
+
+    if (currentRestricted) {
+        return (
+            <div
+                data-testid={props.id}
+                className='form-group'
+            >
+                <div className={'control-label ' + leftCol}>
+                    <LabelRow>
+                        <span>{props.label}</span>
+                        <UpgradePill>{'Enterprise feature'}</UpgradePill>
+                    </LabelRow>
+                </div>
+                <div className={rightCol}>
+                    <LeftBox>
+                        <Title>
+                            <FormattedMessage
+                                defaultMessage={'Use your own WebRTC service for calls and media processing'}
+                            />
+                        </Title>
+                        <VerticalSpacer size={8}/>
+                        <Text>
+                            <FormattedMessage
+                                defaultMessage={'Real-time communication daemon is a service built to offload calls on your own WebRTC services in order to efficiently support scalable and secure deployments. <featureLink>Learn more about this feature</featureLink>.'}
+                                values={{
+                                    featureLink: (text: string) => (
+                                        <a
+                                            href='TODO'
```

Comment:
This link is needed

Reply:
I am thinking either we link to the self-hosted documentation or directly https://github.com/mattermost/rtcd.

Reply:
Sure, we could do that unless we have something more specific.  
@justinegeffen Do you have specific links that you'd like (for this and the other two below)?

Reply:
using: https://docs.mattermost.com/configure/calls-deployment.html